core/src/main/java/org/apache/calcite/sql/validate/DelegatingScope.java

```diff
@@ -542,6 +543,208 @@ private int worstKind(Path path) {
     }
   }
 
+
+  /**
```

Comment:
I'm comparing this to `@Override public SqlQualified fullyQualify(SqlIdentifier identifier)` in the same file